service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/LocalAuthorityEmailContentProvider.java

```diff
@@ -5,26 +5,34 @@
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.notify.BaseCaseNotifyData;
+import uk.gov.hmcts.reform.fpl.model.notify.sdo.SDONotifyData;
 import uk.gov.hmcts.reform.fpl.service.email.content.base.StandardDirectionOrderContent;
 
-import java.util.Map;
-
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCallout;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class LocalAuthorityEmailContentProvider extends StandardDirectionOrderContent {
     private final LocalAuthorityNameLookupConfiguration config;
 
-    public Map<String, Object> buildLocalAuthorityStandardDirectionOrderIssuedNotification(CaseData caseData) {
-        return super.getSDOPersonalisationBuilder(caseData)
-            .put("title", config.getLocalAuthorityName(caseData.getCaseLocalAuthority()))
+    public SDONotifyData buildLocalAuthorityStandardDirectionOrderIssuedNotification(CaseData caseData) {
```

Comment:
```suggestion
    public SDONotifyData buildStandardDirectionOrderIssuedNotification(CaseData caseData) {
```
we already know it is for the LA from the content provider